src/pages/main.js

```diff
@@ -0,0 +1,9 @@
+import Form from '../components/Form';
+
+export default function Main() {
+	return (
+		<>
+			<Form />
```

Comment:
Instead of a component, you should write your code for the component on this page (which you should call form.js), you can make a component out of this, when you are in the position of wanting to reuse the form :)